### ISSUE TITLE
SDK needs to support apps with multiple URL schemes

### ISSUE THREAD

**Issue by postmechanical** [BranchMetrics/Branch-iOS-SDK#51]
Presently the SDK incorrectly assumes that the Branch URL scheme is the first URL scheme in an app's Info.plist. Docs need to be updated to indicate this for apps adding Branch support that may likely already have one or more URL schemes already defined.

**Comment by shortstuffsushi** [comment]
It might also be good to just allow applications to specify their scheme rather than just assuming that the presence of any scheme implies that it's a Branch scheme.

**Comment by qinweigong** [comment]
Thanks for the feedbacks @postmechanical @shortstuffsushi 
I will make changes accordingly.

**Comment by shortstuffsushi** [comment]
Hey @postmechanical. Sorry it's taken so long, but the pull request referencing this issue should address this. Can you take a look and see if it covers what you're looking for?

**Comment by postmechanical** [comment]
@shortstuffsushi that PR looks good from my perspective. Thanks!

**Comment by shortstuffsushi** [comment]
Great! I'll close this now then, feel free to post back with any additional issues you come across, or contact support@branch.io for more help.

**Comment by shortstuffsushi** [comment]
Hi @postmechanical. Just wanted to follow up and let you know that this change was reverted. Upon further discussion, it turns out that the URI Scheme is really only used once while the developer is configured the SDK in order to prefill the URI Scheme on the Branch Dashboard. With this in mind, we decided to remove the extra complexity added by the change in #73, and just leave it as is, with its "best guess" logic. Since we haven't had any issues reported with that, we think that it is sufficient for now. That said, I am updating the readme to better indicate how, where, and when that is used.
